Documentation/release-notes/5111.md

```diff
@@ -0,0 +1,6 @@
+#### Application and library build and deployment
+
+  * [Developer Community 1184573](https://developercommunity.visualstudio.com/content/problem/1167082/xamarin-android-falsely-saying-that-resource-does.html):
+    Xamarin Android falsely saying that Resource does not exist in the current context (CS0103).
+    Duplicate `attr` elements in `declare-styleable`  styles were causing the DesignTime Build
+    system to crash. As a result the `Resource.designer.cs` was not being updated.
```

Comment:
To facilitate collation into the final release notes, this can use the phrase "design-time build"  (the phrase "[design-time build](https://www.bing.com/search?q=%2B"design-time+build"+site%3Adocs.microsoft.com)" is much more common than "[designtime build](https://www.bing.com/search?q=%2B"designtime+build"+site%3Adocs.microsoft.com) on docs.microsoft.com)" and use italics for the file name to match the, admittedly somewhat interesting, convention of italics for file names from the [docs guidelines](https://github.com/MicrosoftDocs/xamarin-docs/blob/c7d82b0684b1c26cdcbc16766b636894a64a80fd/contributing-guidelines/template.md#text-styling).

All together:

```markdown
- [Developer Community 1184573](https://developercommunity.visualstudio.com/content/problem/1167082/xamarin-android-falsely-saying-that-resource-does.html):
  Xamarin Android falsely saying that Resource does not exist in the
  current context (CS0103).  Duplicate `attr` elements in
  `declare-styleable`  styles were causing the design-time build system
  to crash. As a result, the _Resource.designer.cs_ file was not being
  updated.
```